galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -121,3 +121,13 @@ messages:
        - name: annotation
          type: string
 
+  - name: "ConflictingMeshGatewayVirtualServiceHosts"
+    code: IST0109
+    level: Error
+    description: "Conflicting hosts on VirtualServices associated with mesh gateway"
```

Comment:
When you get a chance, please also see about creating some docs for this message (as described here: https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/README.md#6-write-a-user-facing-documentation-page). 

We don't need to do that before shipping this PR, but let's do it before closing the related issue.

Reply:
I was going to do that, so thanks for the note.